scripts/commands.py

```diff
@@ -183,7 +183,7 @@ def commit_memory(string):
 
 
 def delete_memory(key):
-    if key >= 0 and key < len(mem.permanent_memory):
+    if key >= -len(mem.permanent_memory) and key < len(mem.permanent_memory):
```

Comment:
Why did you change this off 0? That doesn't make sense to me; especially since now the memory is twice the size of mem.permanent.memory.

If the AI is returning a negative number, I imagine that it wants to go "back" a couple spots from where it currently is... I'm not sure we have that information available though. Sticking it at the index -4 or something doesn't make sense though, since it's not what a reasonable person/the AI would intend.

Reply:
In Python `a[-1]` for a list is the last position in the array.